tests/test_query.py

```diff
@@ -1240,3 +1240,12 @@ def test_query_datetimes_second(self):
     def test_empty_queryset_bool_converts_to_false(self):
         qs = MockSet()
         assert not bool(qs)
+
+    def test_empty_query(self):
+        car1 = Car(id=101)
+        car2 = Car(id=102)
+
+        mockset = MockSet(car1, car2)
+        self.assertEqual(mockset.count(), 2)
+        self.assertEqual(mockset.filter(Q()).count(), 2)
+        self.assertEqual(mockset.filter(Q(id=2)).count(), 1)
```

Comment:
`AssertionError: 0 != 1`

What is the intention of the last assert?

Btw you can use the same test command as travis from [tox.ini](https://github.com/stphivos/django-mock-queries/blob/master/tox.ini#L31) to verify it works:
```
py.test django_mock_queries/ tests/ --cov-report term-missing --cov=django_mock_queries --flake8
```

Reply:
I removed this,
isn't necessary